components/pedigree/resources/src/main/resources/pedigree/view/personGroupVisuals.js

```diff
@@ -26,14 +26,6 @@ define([
             this.setNumPersons(node.getNumPersons());
         },
 
-        generateHoverbox: function(x, y) {
-            if (editor.isReadOnlyMode()) {
-                return new ReadOnlyHoverbox(this.getNode(), x, y, this.getGenderGraphics());
-            } else {
-                return new PersonGroupHoverbox(this.getNode(), x, y, this.getGenderGraphics());
```

Comment:
I think with this change PersonGroupHoverbox is never used. It may not be a wrong thing in itself, but then we don't need the corresponding class/file. I need a bit more time to read the code, requesting a change now to make sure it does not get merged until checked